modules/network-observability-operator-install.adoc

```diff
@@ -11,21 +11,28 @@ You can install the Network Observability Operator using the {product-title} web
 
 * Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6]. 
 
+[NOTE]
+====
+Your `LokiStack` name and `loki` name must be the same.
```

Comment:
lokistack name should be loki

Reply:
What if the user doesn't want to name it's instance `loki` ?

```suggestion
This documentation assume that your `LokiStack` instance name is `loki`. Using a different name would need additionnal configuration.
```

Reply:
@Amoghrd Changed the capitalization for your review. Thanks!

Reply:
@tmalove the point here was not the `capitalization` but the sentence itself. WDYT about my suggestion above ?